Makefile

```
@@ -85,13 +85,18 @@ infra-update-app-database-roles:
 	@:$(call check_defined, ENVIRONMENT, the name of the application environment e.g. "prod" or "staging")
 	./bin/create-or-update-database-roles.sh $(APP_NAME) $(ENVIRONMENT)
 
-
 infra-update-app-service:
 	# APP_NAME has a default value defined above, but check anyways in case the default is ever removed
 	@:$(call check_defined, APP_NAME, the name of subdirectory of /infra that holds the application's infrastructure code)
 	@:$(call check_defined, ENVIRONMENT, the name of the application environment e.g. "prod" or "staging")
 	./bin/terraform-init-and-apply.sh infra/$(APP_NAME)/service $(ENVIRONMENT)
 
+infra-monitoring-secrets:
+	# APP_NAME has a default value defined above, but check anyways in case the default is ever removed
+	@:$(call check_defined, ENVIRONMENT, the name of the application environment e.g. "prod" or "staging")
+	@:$(call check_defined, URL, endpoint integration URL e.g. "Splunk-On-Call" or "Pagerduty")
```

nit: description should say something about "incident management service", like "incident management service integration URL"

another nit: the examples provided aren't actually URLs.

```suggestion
	@:$(call check_defined, URL, incident management service (PagerDuty or VictorOps) integration URL)
```

---

done